### PR TITLE
Fix demo issue caused by codegrepper extension

### PR DESCRIPTION
**Pull request by k1ng440**
This pull request to resolve [#719](https://github.com/quick-lint/quick-lint-js/issues/719)

### CODE REVIEW

File: website/public/demo/index.ejs.html [hunk 1]
```diff
@@ -49,7 +50,7 @@
       }
       #shadow-code-input {
         z-index: 1;
-        position: absolute;
+        position: absolute !important;
```
Comment (strager) [inline website/public/demo/index.ejs.html:53]: Add a comment explaining the reason for `!important`.

File: website/public/demo/index.ejs.html [hunk 2]
```diff
@@ -11,6 +11,7 @@
       main {
         position: relative;
       }
+
```
Comment (strager) [inline website/public/demo/index.ejs.html:14]: Nit: Undo

File: website/wasm/README.md [hunk 3]
```diff
@@ -24,3 +25,4 @@ To manually test this package, [build](#Building), then run `cli.js`:
 [Ninja]: https://ninja-build.org/
 [Node.js]: https://nodejs.org/
 [emscripten-sdk]: https://emscripten.org/docs/getting_started/downloads.html
+
```
Comment (strager) [inline website/wasm/README.md:28]: Nit: Undo

**Review by strager — CHANGES_REQUESTED** [review]

**Review by strager — APPROVED** [review]

**Comment by k1ng440** [comment]
I have read and hereby agree to quick-lint-js' Contributor License Agreement (CLA-v1.md).

**Pull request merged**